src/main/java/com/ksm/wstbackoffice/controller/AddressController.java

```diff
@@ -1,6 +1,7 @@
 package com.ksm.wstbackoffice.controller;
 
-import com.ksm.wstbackoffice.entity.Address;
+import com.ksm.wstbackoffice.dto.AddressDto;
+import com.ksm.wstbackoffice.entity.AddressEntity;
```

Comment:
We were talking that controllers should not know anything about the DAO layer. And AddressEntity is a DAO class.

Reply:
done